### DISCUSSION TITLE
Question: how to enable html?

### DISCUSSION THREAD

**Discussion by Venegrad** [nuxt-modules/i18n#2161] [Q&A] [answered]
### Describe the feature

I cant use html in my i18n files, and have issue cuz i cant use html, how i can enable it, cuz in nuxt 2 it was be enabled by default.

My config:

```
i18n: {
    locales: [
      {
        code: 'ua',
        file: 'ua-UA.js',
        iso: 'ua-UA',
        name: 'Ukranian'
      }
    ],
    experimental: { 
      jsTsFormatResource: true 
    },
    detectBrowserLanguage: {
      useCookie: true,
      cookieKey: 'i18n_redirected',
      redirectOn: 'root',  // recommended
    },
    lazy: true,
    langDir: 'lang/',
    defaultLocale: 'ua',
    vueI18n: './i18n.config.js' 
  },
```

### Additional information

- [ ] Would you be willing to help implement this feature?
- [ ] Could this feature be implemented as a module?

### Final checks

- [X] Read the [contribution guide](https://nuxt.com/docs/community/contribution) (The contribution guideline of nuxt-modules/i18n is compliant with Nuxt too).
- [X] Check existing [discussions](https://github.com/nuxt-modules/i18n/discussions) and [issues](https://github.com/nuxt/nuxt/issues).

**Comment by kazupon** [comment]
This is a question, so I'll move to github discussions

**Reply by kazupon** [reply, marked as answer]
You can configure `precompile` option.
About details here:
https://v8.i18n.nuxtjs.org/options/misc#precompile

```ts
{
  precompile: {
    strictMessage: false
  }
}
```
